bin/lint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+find xml -name 'issue*.xml' -print0 | \
+xargs -0 -P $(getconf _NPROCESSORS_ONLN) -n 128 xmllint --noout --nowarning --dtdvalid xml/lwg-issue.dtd
```

Comment:
This looks horrible to me with the pipeline split onto two lines with the same indentation; is there an idiomatic way to format it?

Reply:
And if you want shorter lines, you could even break between the xargs options and the command it runs, with another level of indentation:
```sh
find xml -name 'issue*.xml' -print0 \
  | xargs -0 -P $(getconf _NPROCESSORS_ONLN) -n 128 \
    xmllint --noout --nowarning --dtdvalid xml/lwg-issue.dtd
```